openmc/cppapi/cppapi.cpp

```diff
@@ -0,0 +1,46 @@
+#include <pybind11/pybind11.h>
+#include "openmc/capi.h" 
+#include <string>
+#include <vector>
+#include <pybind11/stl.h>
+#include <algorithm>
```

Comment:
Group together and separate includes by 1. local imports, 2. third-party, 3. standard library:
```C++
#include "openmc/capi.h"

#include <pybind11/pybind11.h>
#include <pybind11/stl.h>

#include <algorithm>
#include <string>
#include <vector>
```

Also, does pybind11.h not include stl.h?